.github/workflows/aks_deploy.yml

```
@@ -102,23 +102,33 @@ jobs:
   deploy_sandbox:
     needs: [deploy_staging]
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/main'
     environment:
       name: sandbox
 
     steps:
       - uses: actions/checkout@v3
-      - run: |
-          echo "deploying sandbox"
+
+      - uses: ./.github/actions/deploy-environment-to-aks
+        id: deploy
+        with:
+          environment: sandbox
+          docker-image: ${{ needs.docker.outputs.image }}
```

This will probably not work since the deploy_sandbox job doesn't "need" the docker job

---

Ah - well spotted. So looking [at the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idoutputs) this could simply be this (I think):

```suggestion
          docker-image: ${{ docker.outputs.image }}
```

---

My assumption was incorrect, repushed in a way that passes the `docker-image` along